internal/cloudsql/refresh_test.go

```diff
@@ -79,24 +81,65 @@ func TestRefresh(t *testing.T) {
 	if wantPrivateIP != gotIP {
 		t.Fatalf("metadata IP mismatch, want = %v, got = %v", wantPrivateIP, gotIP)
 	}
-	gotPSC, ok := rr.addrs[PSC]
+	gotDNS, ok := rr.addrs[PSC]
```

Comment:
this looks like a typo...
```suggestion
	gotPSC, ok := rr.addrs[PSC]
```

Reply:
Thanks, Jack. I just renamed it to `gotPSCDNS` to make it clear that the DNS is from PSC addresses instead of the direct DNS name of the connection info `rr`.

Fixing the failure introduced by the new test now. Thanks.